src/Uno.UI.Wasm/ts/Windows/Devices/Enumeration/Internal/Providers/Midi/MidiDeviceClassProvider.ts

```diff
@@ -0,0 +1,25 @@
+﻿namespace Uno.Devices.Enumeration.Internal.Providers.Midi {
+	export class MidiDeviceClassProvider {		
+		public static findDevices(findInputDevices: boolean): string {
+			var result = "";
+			var midi = Uno.Devices.Midi.Internal.WasmMidiAccess.getMidi();
+			if (findInputDevices) {
+				midi.inputs.forEach((input: WebMidi.MIDIInput, key: string) => {
+					var inputId = input.id;
+					var name = input.name;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'name' is never reassigned; use 'const' instead of 'var'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)